src/node_crypto.cc

```diff
@@ -1536,8 +1536,9 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
                                     String::kNormalString, mem->length));
       (void) BIO_reset(bio);
 
-      BN_ULONG exponent_word = BN_get_word(rsa->e);
-      BIO_printf(bio, "0x%lx", exponent_word);
+      unsigned long exponent =  // NOLINT(runtime/int)
+        static_cast<unsigned long>(BN_get_word(rsa->e));  // NOLINT(runtime/int)
```

Comment:
This is not correct per se on 32 bits architectures where `long` is 32 bits and BN_ULONG is a 64 bits `unsigned long long`.